solarwinds_apm/configurator.py

```diff
@@ -98,39 +99,41 @@ def _configure(self, **kwargs: int) -> None:
         apm_txname_manager = SolarWindsTxnNameManager()
         apm_fwkv_manager = SolarWindsFrameworkKvManager()
         apm_config = SolarWindsApmConfig()
-        oboe_api = apm_config.oboe_api
+        # oboe_api = apm_config.oboe_api
 
         # Reporter may be no-op e.g. disabled, lambda
-        reporter = self._initialize_solarwinds_reporter(apm_config)
+        # reporter = self._initialize_solarwinds_reporter(apm_config)
         self._configure_otel_components(
             apm_txname_manager,
             apm_fwkv_manager,
             apm_config,
-            reporter,
-            oboe_api,
+            None,
+            None,
+            # reporter,
+            # oboe_api,
         )
 
-        if apm_config.is_lambda:
-            logger.debug("No init event in lambda")
-            return
+        # if apm_config.is_lambda:
+        #     logger.debug("No init event in lambda")
+        #     return
 
         # Report reporter init status event after everything is done.
-        init_event = self._create_init_event(reporter, apm_config)
-        if init_event:
-            self._report_init_event(reporter, init_event)
-        else:
-            logger.error(
-                "There was an issue with generating the reporter init message."
-            )
+        # init_event = self._create_init_event(reporter, apm_config)
+        # if init_event:
+        #     self._report_init_event(reporter, init_event)
+        # else:
+        #     logger.error(
+        #         "There was an issue with generating the reporter init message."
+        #     )
 
     # TODO update pylint disable when drop py38 support
     def _configure_otel_components(
         self,
         apm_txname_manager: SolarWindsTxnNameManager,
         apm_fwkv_manager: SolarWindsFrameworkKvManager,
         apm_config: SolarWindsApmConfig,
-        reporter: "Reporter",
-        oboe_api: "OboeAPI",
+        reporter,
+        oboe_api,
```

Comment:
Nit: could we do this here and similarly for the others?
```suggestion
        reporter: Any = None,
        oboe_api: Any = None,
```

I thought CodeQL would have more to say about these, but it might not run unless PR is pointed at `main`.